docs/Configuration.rst

```diff
@@ -5981,7 +6036,166 @@ data is stored in the database.
+              When it gets mined, the Job ``ContractTransactionVerifierJob`` will detect it and will store again the real transaction status coming from blockchain and send notifications.
+
+
+Wallet types
+~~~~~~~~~~~~
+
+* **Admin wallet**
+
+  The *Admin wallet* is created automatically during first startup of the eXo Platform server.
+
+  In order to be able to issue transactions on the blockchain using `Web3j<https://web3j.io/>`__ server side, the private key of the *Admin wallet* is stored (in encrypted form) in the internal database.
+  The private key is encrypted by combining two passwords:
+
+     -  A first password is read from a keystore file ( see :ref:`Updating password encryption key <UpdatePasswordEncryptionKey>` )
+
+     .. warning:: The generated keystore file under `gatein/conf/codec` MUST be backed up as a data folder, because it contains a key file that is used to decrypt stored wallets private keys. If it's lost, all wallets private keys will be lost and consequently, all funds would be lost and unrecoverable.
+
+     -  A second password that can be configured in properties:
+
+    .. code-block:: jproperties
+
+          exo.wallet.admin.key=changeThisKey
+
+
+    .. warning:: The password can't be changed once the platform is started for the first time. In fact, this password will be used to encrypt the *Admin wallet*'s private key that will be stored in database.
+        If its value is modified after server startup, the private key of admin wallet won't be decrypted anymore, preventing all administrative operations.
+
+
+  The *Admin wallet* can be used by */platform/rewarding* group members to initialize other wallets.
+  The private key of the admin wallet cannot be accessed by any user to avoid exposing its funds to an unauthorized person.
+
+* **User wallet**
+
+  By default, any user of the platform can create his own wallet. A property is added to restrict access to wallet application to a group of users:
+
+    .. code-block:: jproperties
+
+          # Example of expressions
+          
+          # Any user
+          exo.wallet.accessPermission=
+          # All connected users of the platform
+          exo.wallet.accessPermission=/platform/users
+          # All administrators of the platform
+          exo.wallet.accessPermission=*:/platform/administrators
+          # All members of space associated to group internal_space
+          exo.wallet.accessPermission=member:/spaces/internal_space
+
+* **Space wallet**
+
+  Space managers can add the wallet application in their space a wallet for a space using :ref:`Managing space applications <ManagingSpaceApplication>` UI:
+
+  |SpaceAddWallet|
+
+  Space wallets can be managed by one or multiple space manager. In fact, the space wallet is similar to user wallet. To use it, it must be activated by an admin. The operation is carried out by the *Admin Wallet* account.
```

Comment:
```suggestion
  Space wallets can be managed by one or multiple space manager. In fact, the space wallet is similar to user wallet. To use it, it must be activated by an admin. The operation is carried out by the *Admin Wallet* account.
```

it is not clear if space managers have different privileges than regular space members. Can all space managers send space wallet funds ?

Reply:
Space wallets can be managed by one or several space manager. In fact, by default the space wallet private key is stored in eXo internal database. So all managers has access to it and so they will automatically download it from server. But the the private key still encrypted using a password, that only wallet creator (that is one of space managers) owns it and could share with other managers to "send" funds from space wallet.